pkg/api/norman/customization/cluster/validator.go

```diff
@@ -37,6 +37,10 @@ type Validator struct {
 	CisBenchmarkVersionLister     v3.CisBenchmarkVersionLister
 }
 
+var (
+	availableVersions = []string{"1.17", "1.16", "1.15"}
```

Comment:
These should be read from some system settings/config. Do we need to hardcode here?

Reply:
Thinking of removing this entirely since this is already done by EKS api and keeping it prevents us from updating version support without a rancher release.